_studio/shared/umc/codec/h264_enc/include/umc_h264_common.h

```diff
@@ -19,7 +19,6 @@
 // SOFTWARE.
 
 #include "umc_defs.h"
-#if defined(UMC_ENABLE_H264_VIDEO_ENCODER)
```

Comment:
Why this was removed rather than changed to MFX_ENABLE_...?

Reply:
It causes compilation error in mfx_h264_enc_common_hw.cpp (UMC_H264_ENCODER’ has not been declared) without enable option for h264 encoder patch. MFX_ENABLE_H264_VIDEO_ENCODE has to be defined somewhere.

Reply:
Ok, let's wait for the #516 then.